src/crafting.cpp

```diff
@@ -2104,6 +2106,81 @@ ret_val<bool> Character::can_disassemble( const item &obj, const inventory &inv
     return ret_val<bool>::make_success();
 }
 
+item_location Character::create_in_progress_disassembly( item_location target )
+{
+    const auto &r = recipe_dictionary::get_uncraft( target->typeId() );
+    item &temp = *target.get_item();
+
+    item new_disassembly( &r, temp );
+    target.remove_item();
+
+    item_location disassembly_in_world;
+
+    if( !has_two_arms() ) {
+        disassembly_in_world = set_item_map_or_vehicle( *this, pos(), new_disassembly );
+    } else if( !has_wield_conflicts( new_disassembly ) ) {
+        if( cata::optional<item_location> it_loc = wield_craft( *this, new_disassembly ) ) {
+            disassembly_in_world = *it_loc;
+        }  else {
+            // This almost certianly shouldn't happen
```

Comment:
```suggestion
            // This almost certainly shouldn't happen
```
Is a debug message indicated here?